release-notes/supported-os.json

```diff
@@ -0,0 +1,121 @@
+                    "eos-date": "2022-10-01"
+                },
+                {
+                    "cycle": "22.04",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2024-09-30"
+                },
+                {
+                    "cycle": "22.10",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2023-07-20"
+                },
+                {
+                    "cycle": "23.04",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2024-01-20"
+                },
+                {
+                    "cycle": "23.10",
+                    "supported-channels": ["6.0", "7.0", "8.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2024-07-01"
+                },
+                {
+                    "cycle": "24.04",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2029-07-01"
+                }
+            ]
+        },
+        {
+            "name": "iOS",
+            "lifecycle-policy": "https://support.apple.com/ios",
+            "last-updated": "2023-02-28",
+            "upcoming-eos": {
+                "cycle": "15",
+                "expected-eos-date": "2024-10-01"
+            },
+            "cycles": [
+                {
+                    "cycle": "10",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2017-09-26"
+                },
+                {
+                    "cycle": "11",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2018-10-08"
+                },
+                {
+                    "cycle": "12",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2023-01-23"
+                },
+                {
+                    "cycle": "13",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2020-09-16"
+                },
+                {
+                    "cycle": "14",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2021-10-01"
+                },
+                {
+                    "cycle": "15",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "expected-eos-date": "2024-10-01"
```

Comment:
`eol.date` only shows `eol: false` in this case as there is no announcement from Apple. There is only the expectation that it will be killed after the next announcements in september. Helpful for having as trigger, but totally speculatative. So not sure if we should add it in the first place.